src/live_view.cpp

```diff
@@ -40,76 +28,47 @@ void live_view::init( int const start_x, int const start_y, int const w, int con
     hide();
 }
 
-void live_view::show( const int x, const int y, const visibility_variables &cache )
+int live_view::draw()
 {
-    if( !enabled || !w_live_view ) {
-        return;
+    if( !enabled ) {
+        return 0;
     }
 
-    hide( false ); // Clear window if it's visible
+    werase( *this );
+
+    // -1 for border. -1 because getmaxy() actually returns height, not y position.
+    const int last_line = height - 2;
+    const visibility_variables &cache = g->m.get_visibility_variables_cache();
+    bottom_border_line = START_LINE;
+    w_live_view->height = height;
+    g->print_all_tile_info( mouse_position, *this, START_COLUMN, bottom_border_line,
+                            last_line, false, cache );
 
+    w_live_view->height = std::max( bottom_border_line + 1, 11 );
```

Comment:
You can't change a window size that way. `ncurses` windows don't have this property.